<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderManagerForm</class>
 <widget class="QWidget" name="OrderManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="toolVerticalLayout">
       <item>
        <widget class="QGroupBox" name="searchGroupBox">
         <property name="title">
          <string>Search</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="showAllPushButton">
            <property name="text">
             <string>Show All</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="searchHorizontalLayout">
            <item>
             <widget class="QComboBox" name="searchComboBox">
              <item>
               <property name="text">
                <string>ID</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Date</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Client</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Product</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="searchLineEdit"/>
            </item>
            <item>
             <widget class="QDateEdit" name="searchDateEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="searchPushButton">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="inputGroupBox">
         <property name="title">
          <string>Add / Modify</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="orderFormLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="idLabel">
              <property name="text">
               <string>ID</string>
              </property>
              <property name="buddy">
               <cstring>idLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="idLineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="clientLabel">
              <property name="text">
               <string>Client</string>
              </property>
              <property name="buddy">
               <cstring>clientLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="clientLineEdit">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="inputClientPushButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resource.qrc">
                  <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="productLabel">
              <property name="text">
               <string>Product</string>
              </property>
              <property name="buddy">
               <cstring>productLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="productLineEdit">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="inputProductPushButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resource.qrc">
                  <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="dateLabel">
              <property name="text">
               <string>Date</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDateEdit" name="dateEdit"/>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="quantitySpinBox">
              <property name="maximum">
               <number>9999999</number>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="quantityLabel">
              <property name="text">
               <string>Quantity</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="addHorizontalLayout">
            <item>
             <widget class="QPushButton" name="addPushButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="modifyPushButton">
              <property name="text">
               <string>Modify</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cleanPushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/images/erase.png</normaloff>:/images/erase.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <layout class="QVBoxLayout" name="listVerticalLayout">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Client</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Product</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Total</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="detailsGroupBox">
         <property name="title">
          <string>Details</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTreeWidget" name="clientTreeWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Phone Number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Address</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QTreeWidget" name="productTreeWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <column>
             <property name="text">
              <string notr="true">ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Type</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Unit Price</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Quntities in stock</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
